src/test/regress/expected/isolation_multiuser_locking_0.out

```diff
@@ -0,0 +1,358 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1-no-connection-cache s2-no-connection-cache s1-begin s2-begin s2-reindex s1-insert s2-commit s1-commit
+create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+step s1-no-connection-cache:
+ SET citus.max_cached_conns_per_worker to 0;
+
+step s2-no-connection-cache:
+ SET citus.max_cached_conns_per_worker to 0;
+
+step s1-begin:
+ BEGIN;
+ SET ROLE test_user_1;
+
+step s2-begin:
+ BEGIN;
+ SET ROLE test_user_2;
+
+step s2-reindex:
+ REINDEX TABLE test_table;
+
+ERROR:  permission denied for table test_table
+step s1-insert:
+ UPDATE test_table SET column2 = 1;
+
+step s2-commit:
+ COMMIT;
+
+step s1-commit:
+ COMMIT;
+
+
+starting permutation: s1-no-connection-cache s2-no-connection-cache s1-grant s1-begin s2-begin s2-reindex s1-insert s2-insert s2-commit s1-commit
+create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+step s1-no-connection-cache:
+ SET citus.max_cached_conns_per_worker to 0;
+
+step s2-no-connection-cache:
+ SET citus.max_cached_conns_per_worker to 0;
+
+step s1-grant:
+ SET ROLE test_user_1;
+ GRANT ALL ON test_table TO test_user_2;
+
+step s1-begin:
+ BEGIN;
+ SET ROLE test_user_1;
+
+step s2-begin:
+ BEGIN;
+ SET ROLE test_user_2;
+
+step s2-reindex:
+ REINDEX TABLE test_table;
+
+step s1-insert:
+ UPDATE test_table SET column2 = 1;
+ <waiting ...>
```

Comment:
this seems a bit suspicious as we were throwing a permission-check error in the older test output

Reply:
I suppose that because of this commit:
 
[git.postgresql.org Git - postgresql.git/commitdiff](https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=ecb0fd337)

If you think that it is unrelated, ı Will looked into that more.